spec/system/candidate_interface/candidate_entering_disability_info.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering their disability information' do
+  include CandidateHelper
+
+  scenario 'Candidate submits their disability information' do
+    given_i_am_not_signed_in
+    and_i_visit_the_training_with_a_disability_page
+    then_i_should_see_the_homepage
```

Comment:
Perhaps we can skip the 3 steps above, the auth is tested in a couple of places already.